Cargo.toml

```diff
@@ -17,13 +17,19 @@ azure-devops = { project = "jonhoo/jonhoo", pipeline = "openssh", build = "23" }
 codecov = { repository = "jonhoo/openssh-rs", branch = "master", service = "github" }
 maintenance = { status = "experimental" }
 
+[features]
+mux_client = ["tokio/rt", "once_cell", "tokio-pipe", "openssh-mux-client"]
```

Comment:
I have renamed the feature into `native_mux` since `-` cannot be used in the name of rust `struct`/`enum`.

Reply:
The convention is to still use `-` in feature names (see, for example, (`ripgrep`'s features](https://docs.rs/crate/regex/1.5.4/features). The fact that you can't use `-` in type names doesn't really matter, since Rust uses CamelCase names anyway, and so there's neither a `-` nor a `_` in there.

Reply:
OK, I've changed it to `native-mux`.